components/Carousel/Carousel.js

```diff
@@ -1,14 +1,40 @@
 class Carousel {
+  constructor() {
+    this.rightButton = document.querySelector(".right-button")
+    this.leftButton = document.querySelector(".left-button")
+    let boxes = document.querySelectorAll(".carousel")
+    this.selectedBox = boxes[0].
```

Comment:
You're on the right path, try to add a counter here to keep track of indexes, then use those indexes to display the respective image